editor/src/utils/deep-equality.ts

```diff
@@ -358,6 +358,72 @@ export function combine9EqualityCalls<A, B, C, D, E, F, G, H, I, X>(
     }
   }
 }
+
+export function combine10EqualityCalls<A, B, C, D, E, F, G, H, I, J, X>(
```

Comment:
So much equality!

Reply:
a few weeks ago I spent a day trying to use new typescript type features to annotate a combineNEqualityCalls function but I failed